tests/test_optimize.py

```diff
@@ -89,22 +88,13 @@
 
 
 @qml.qnode(qml.device("default.qubit", wires=1))
-def quant_fun(variables):
+def quant_fun(*variables):
```

Comment:
?

Reply:
Since we do not allow ragged input arrays in tape mode, the input is now passed as positional arguments (e.g., we unpack the ragged array)

Reply:
I think it is just better practice to do the unpacking when calling a function...?